src/corppa/poetry_detection/passim/run_passim.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+#####
+# Script for running passim. Requires Java 8*/11/17 because of Spark dependency.
+#
+# Examples:
+#   env PATH="/opt/homebrew/opt/openjdk@17/bin:$PATH" ./run_passim.sh [input] [output]
+#####
+
+# Spark settings
+export SPARK_LOCAL_IP="127.0.0.1"
+export SPARK_SUBMIT_ARGS='--master local[6] --driver-memory 8G --executor-memory 8G'
+
+IN=$1
+OUT=$2
+
+
+if [ $# -ne 2 ]; then
+  echo "Usage: [input] [output]"
+  exit 1
+fi
+
+# Check for supported java version (via major version number)
```

Comment:
I mean, I did this because the error messages are *very* unclear. And this is a repeated issue for me because I need to run with `env` where I remember to set all the necessary environment variables.

I included this bit when I was trying to figure out why things were crashing, it kept varying because a Java version problem and other MAC-related issues.